snabbdom.js

```diff
@@ -45,10 +46,13 @@ function init(modules) {
   }
 
   function createElm(vnode, insertedVnodeQueue) {
-    var i, data = vnode.data;
+    var i, thunk, data = vnode.data;
```

Comment:
Snabbdom should not know about thunks. The functionality with regards to thunks should be contained in the thunk module. It should be possible.